packages/atlas/src/components/ListItem/ListItemSeparator.tsx

```diff
@@ -0,0 +1,10 @@
+import { SeparatorWrapper } from '@/components/ListItem/ListItem.styles'
+
+export const ListItemSeparator = () => {
+  return (
+    <SeparatorWrapper>
+      <div />
+      <div />
```

Comment:
This is confusing for me, I would opt for creating styled component for each div and style them separately.